api/CSSFontFeatureValuesRule.json

```diff
@@ -26,7 +26,7 @@
           "webview_android": "mirror"
         },
         "status": {
-          "experimental": false,
```

Comment:
In safari actually https://github.com/WebKit/WebKit/blob/main/Source/WebCore/css/CSSFontFeatureValuesRule.idl